### PR TITLE
add twint integration

### PR DESCRIPTION
**Pull request by resslinger**

### CODE REVIEW

**Comment by fjbender** [comment]
Thank you for your contribution! This looks pretty solid. Let's get our pipelines sorted and then review this.

File: Components/Payment/Provider/ActivePaymentMethodsProvider.php
Comment (resslinger) [inline Components/Payment/Provider/ActivePaymentMethodsProvider.php:1]: The issue is addressed and Mollie will fixed it. So this change can be reverted, if the fix is live.

**Comment by resslinger** [comment]
These changes are in production and we already have received twint transactions successfully.

**Comment by resslinger** [comment]
@fjbender @boxblinkracer Any update?

**Comment by fjbender** [comment]
Sorry for the delay here - we have a lot going on in Shopware 6 as well, and @boxblinkracer had a lot of talk to deliver on various Shopware conferences ;)

I still think this is a very solid PR and can be merged as soon as we fixed the pipeline and are gearing up for a new release. I don't expect this to happen in December, though.

Thanks for your understanding!

**Comment by boxblinkracer** [comment]
Hi @resslinger  and @fjbender 
so i've been out a while and into lots of SW6 features
again thanks for this amazing PR
i will now merge it and make sure pipe is green and correct TestRail case IDs are set :)

release is then after christmas beginning next year

**Comment by resslinger** [comment]
Hi @boxblinkracer , is a release date in planning?

**Pull request merged**